Update lab-codespaces.md

haydenrandolph opened the pull request.

haydenrandolph committed:
Update lab-codespaces.md

marc-sensenich commented on lab-codespaces.md:
```diff
@@ -31,7 +31,12 @@ Link: [Workshop repo](https://github.com/BoxBoat-Codespaces/codespaces-workshop)
-The destination should be a Codespaces enabled Org and your company likely is providing one for you. You should prepend the repo name with your username, so if your username is dkoch, your repo should be `dkoch-codespaces-workshop`.
+Create a PAT inside your GitHub profile settings > Developer settings > Personal access tokens. 
+
+The destination should be a **Codespaces enabled Org** and your company likely is providing one for you. You should prepend the repo name with your username, so if your username is dkoch, your repo should be `dkoch-codespaces-workshop`. Below is a screenshot of how to launch Codespaces from the repository. If you do not see the Codespaces selection OR cannot "Create codespace..." reach out to the ...
```
Maybe a note at the end to also ensure they've created it in the organization account and not their personal account? Something like
```
... OR cannot "Create codespace..." ensure that you've created the repository in the Codespaces enabled org or reach out to the Orgs admin . 
```

If you're planning on adding a troubleshooting section later, you can skip this for now

marc-sensenich commented on lab-codespaces.md:
```diff
@@ -31,7 +31,12 @@ Link: [Workshop repo](https://github.com/BoxBoat-Codespaces/codespaces-workshop)
 Create a new repo from the template linked above. Private works fine.
 
-The destination should be a Codespaces enabled Org and your company likely is providing one for you. You should prepend the repo name with your username, so if your username is dkoch, your repo should be `dkoch-codespaces-workshop`.
+Create a PAT inside your GitHub profile settings > Developer settings > Personal access tokens. 
```
How does this differ from the line starting with `First thing we need to do is to make a PAT.` below? Should we change the ordering of the current two paragraphs?

Either way, I'd recommend linking to https://github.com/settings/tokens for ease of navigation

haydenrandolph committed:
updating flow 

included your "...ensure that it was created...." & cleaned up repetitive instructions. 
included link to profile settings
updated workflow to be: create PAT > create Repo & Add PAT > Launch codespaces

marc-sensenich commented on lab-codespaces.md:
```diff
@@ -25,19 +25,24 @@ page_nav:
-## Create a repo, a Personal Access Token (PAT) and launch Codespaces
+## Create a a Personal Access Token (PAT), repo and launch Codespaces
 
+Link: [Profile Settings](https://github.com/settings/tokens)
 Link: [Workshop repo](https://github.com/BoxBoat-Codespaces/codespaces-workshop)
```
Drop these two lines to remove the extra links up top on the page as the flow looks odd with them in there IMO, and then it LGTM

```suggestion
```

<img width="969" alt="image" src="https://user-images.githubusercontent.com/10442429/184195685-1e88afe5-ca63-4b9f-ab36-e88153517307.png">

marc-sensenich commented:
Also @haydenrandolph there's the DCO resolution

haydenrandolph committed:
Update lab-codespaces.md

applying suggestion

Co-authored-by: Marc Sensenich <marc-sensenich@users.noreply.github.com>
